creator-node/src/services/stateMachineManager/stateMonitoring/StateMonitoringQueue.js

```diff
@@ -27,7 +27,7 @@ class StateMonitoringQueue {
       queue: this.queue,
       jobSuccessCallback: this.enqueueJobAfterSuccess,
       jobFailureCallback: this.enqueueJobAfterFailure,
-      processJob: this.processJob
+      processJob: this.processJob.bind(this)
```

Comment:
yea this particular case is because processJob needs to use `this.log()`, but `processJob` is passed as a function that the queue calls later. so it wasn't preserving the `this` context until I bound it